rllib/agents/a3c/a3c_tf_policy.py

```diff
@@ -36,7 +36,7 @@ def __init__(self,
 
 def actor_critic_loss(policy, model, dist_class, train_batch):
     model_out, _ = model.from_batch(train_batch)
-    action_dist = dist_class(model_out, model)
+    action_dist = dist_class(model_out)  # , model)
```

Comment:
Please keep the model parameter, it is used for supporting autoregressive distributions: https://ray.readthedocs.io/en/latest/rllib-models.html#autoregressive-action-distributions

Reply:
All back now.